contrib/hooks/hpc2n_hooks.py

```diff
@@ -0,0 +1,205 @@
+# Hooks for HPC2N site changes.
+#
+# Author: Ake Sandgren, HPC2N
+
+import os
+
+from distutils.version import LooseVersion
+from easybuild.framework.easyconfig.format.format import DEPENDENCY_PARAMETERS
+from easybuild.tools.filetools import apply_regex_substitutions
+from easybuild.tools.build_log import EasyBuildError
+from easybuild.tools.modules import get_software_root
+from easybuild.tools.systemtools import get_shared_lib_ext
+
+# Add/remove dependencies and/or patches
+# Access to the raw values before templating and such.
+def parse_hook(ec):
```

Comment:
add `*args` and `**kwargs`, as recommended in the documentation?

Reply:
perhaps...